### DISCUSSION TITLE
Reviewers Wanted!

### DISCUSSION THREAD

**Discussion by jinek** [Consolonia/Consolonia#22] [Q&A] [answered]
Please, let me know if you could spend some time (any time) reviewing the code.
Absence of reviewers currently blocks me from merging and doing normal regular branching.
You can even just jump to PRs and review something you'd like to.
I don't expect any  heavy discussions there since project is at very beginning, but even brief look would work.

Little bit later I will setup code styling/static analysis and ofcoz CI/tests.

**Comment by tomlm** [comment, marked as answer]
Hey Evgeny, I am available!
